recipes/pypromice/meta.yaml

```diff
@@ -1,69 +0,0 @@
-{% set name = "pypromice" %}
-{% set version = "1.3.1" %}
```

Comment:
Please don't delete other people's packages from the recipes directory. It causes merge conflicts in our main branch. We have a bot which deletes packages as they are converted.

Reply:
Will close and resubmit the PR

Reply:
You can use `git rebase -i`

Reply:
@carterbox Is this still needed?